NUnitTestMarsRover/HeadingTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using NUnit.Framework;
+using MarsRover;
+
+namespace NUnitTestMarsRover
```

Comment:
No change requested here.  However, like in my other comment, this should be split into 4 test classes to maintain a 1-to-1 relationship between production code files and test files.

i.e. `NorthHeadingTests.cs`, `SouthHeadingTests.cs`, etc.

Then in your `[SetUp]` function you could just have:

```cs
private NorthHeader _subject;

[SetUp]
public void SetUp()
{
    _subject = new NorthHeading();
}
```